packages/ckeditor5-emoji/docs/features/emoji.md

```diff
@@ -134,6 +130,29 @@ The feature can be configured via the {@link module:emoji/emojiconfig~EmojiConfi
 		.catch( /* ... */ );
 	```
 
+### Emoji's availability and appearance
+
+The availability of the emoji depends on the operating system. Different systems will have different Unicode support. You can decide that you want to lower the amount of newer emoji by setting a lower version of Unicode of the emoji repository (CKEditor hosts v15 and v16). This way the users with newer systems will not be able to use newer emoji. Keep in mind that this only affects the editor's feature. A user will still be able to use the native emoji insertion methods. The availability may also increase with the usage of a custom font. To learn more, read the next paragraph.
+
+If you want to standardize the appearance emojis accross operating systems, please consider using an external font in your integration, such as [Noto Color Emoji](https://fonts.google.com/noto/specimen/Noto+Color+Emoji). In the setup make sure to:
+
+1. Set in the `font-family` for the content, this way the emoji in the editable will use the custom font. For example:
+
+    ```css
+    body {
+    	font-family: 'Lato', 'Noto Color Emoji', sans-serif;
+    }
+    ```
+
+2. Update the `--ck-font-face` variable, so that emoji in the picker and mention use the custom font.
+
+    ```css
+    :root {
+        --ck-font-face: Helvetica, Arial, Tahoma, Verdana, 'Noto Color Emoji';
```

Comment:
Loud thinking: shouldn't we set the `font-face` definition in the following elements?

* https://github.com/ckeditor/ckeditor5/blob/26bf48f62284cffdb1079083c509d3a7d091b386/packages/ckeditor5-emoji/src/emojiutils.ts#L122
*

Reply:
Follow-up sounds like a nice idea.

Reply:
👉 #18029